app/views/devise/sessions/new.html.erb

```diff
@@ -1,26 +1,35 @@
-<h2>Log in</h2>
+<div style="width: 450px;margin: 0 auto; border-radius: 10px; padding: 55px 55px 37px 55px; background-color: rgb(215,215,215);">
```

Comment:
Don't use inline css styles, you already have everything you need in Bootstrap, use bootstrap classes
https://getbootstrap.com/docs/4.3/getting-started/introduction/